python/src/scipp/_math.py

```diff
@@ -124,3 +124,60 @@ def log10(x: VariableLike, *, out: Optional[VariableLike] = None) -> VariableLik
     :returns: Base 10 logarithm of the input.
     """
     return _call_cpp_func(_cpp.log10, x, out=out)
+
+
+def around(x: VariableLike,
+           decimals: int = 0,
+           *,
+           out: Optional[VariableLike] = None) -> VariableLike:
+    """
+    Round to the nearest integer if no decimals is provided, otherwise round to the
+    nearest decimal given.
+
+    Note: if the number being rounded is halfway between two integers/decimals it
+    will round to the nearest even number. For example 1.5 and 2.5 will both round
+    to 2.0, -0.5 and 0.5 will both round to 0.0.
+
+    :param x: Input data.
+    :param decimals: Number of decimal places to round to.
+    :param out: Optional output buffer.
+    :returns: Rounded version of the data passed to the decimals given, if given,
+    else to the nearest integer.
+    """
+    if out is not None:
+        # Copy x to out, if out != x.
+        out[...] = x
+    else:
+        out = x
+    multiplier = 10.0**decimals
+    out = out * multiplier
+    _cpp.round(out, out=out)
+    out /= multiplier
+    return out
+
+
+# Add an alias like numpy to allow users to call round_ instead of around, as it
+# may be clearer to users.
+round_ = around
```

Comment:
I'm on the fence about this, and `around` naming. I have always wondered how much of this is historic in numpy (from the times when more people used `from package import *`). We have a similar case with `min` and `max`, where we currently break with the numpy naming. In my eyes having multiple names and functions that to similar or identical things is very confusing.

I would like to hear your opinion on this, but for now I would suggest to just have `round` (not `around` or `round_`). If we change our mind about this we can add it for all similar cases in one go.

Reply:
Main reason I moved away from using just "round" was because when it is imported, it hides the standard library python function of the same name. 
That leads to me using the numpy names, I don't like "around" but it's what numpy uses so that's what I used. I actually prefer "round_" just because when I am working I will type "round" and it will autocomplete to the "round_"

Reply:
But numpy *also* provides `round`!?

Reply:
I see it is, but it seems only as a method opposed to a raw function, it is quite possible that I can't see the docs for that function though. 

So we could always support `variable.round` as a method.

Reply:
See `__init__.py` on how we add methods (though in this case it may need to be done slightly differently, since the name of method and free function differ).

Reply:
Added methods! and naming resolved in offline/team discussion :+1:

Decided upon round as it follows the python array API spec: https://data-apis.org/array-api/latest/API_specification/index.html